app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt

```diff
@@ -54,9 +54,11 @@ class ShortcutReceiver @Inject constructor(
         }
 
         GlobalScope.launch(dispatcher.io()) {
-            if (ctaDao.exists(CtaId.USE_OUR_APP) && originUrl == USE_OUR_APP_SHORTCUT_URL) {
```

Comment:
We don't want to check if a user has seen a CTA before so we can compare against the control group.